lib/client-assets.php

```diff
@@ -133,6 +133,224 @@ function gutenberg_override_style( $handle, $src, $deps = array(), $ver = false,
+			'wp-html-entities',
+			'wp-i18n',
+			'wp-is-shallow-equal',
+			'wp-keycodes',
+			'wp-polyfill',
+			'wp-rich-text',
+			'wp-url',
+		),
+		'compose' => array(
+			'lodash',
+			'wp-deprecated',
+			'wp-element',
+			'wp-is-shallow-equal',
+			'wp-polyfill'
+		),
+		'core-data' => array( 'wp-data', 'wp-api-fetch', 'wp-polyfill', 'wp-url', 'lodash' ),
+		'data' => array(
+			'lodash',
+			'wp-compose',
+			'wp-deprecated',
+			'wp-element',
+			'wp-is-shallow-equal',
+			'wp-polyfill',
+			'wp-redux-routine',
+		),
+		'date' => array( 'moment', 'wp-polyfill' ),
+		'deprecated' => array( 'wp-polyfill', 'wp-hooks' ),
+		'dom' => array( 'lodash', 'wp-polyfill', 'wp-tinymce' ),
+		'dom-ready' => array( 'wp-polyfill' ),
+		'edit-post' => array(
+			'jquery',
+			'lodash',
+			'postbox',
+			'media-models',
+			'media-views',
+			'wp-a11y',
+			'wp-api-fetch',
+			'wp-block-library',
+			'wp-blocks',
+			'wp-components',
+			'wp-compose',
+			'wp-core-data',
+			'wp-data',
+			'wp-deprecated',
+			'wp-dom-ready',
+			'wp-editor',
+			'wp-element',
+			'wp-embed',
+			'wp-i18n',
+			'wp-keycodes',
+			'wp-nux',
+			'wp-plugins',
+			'wp-polyfill',
+			'wp-url',
+			'wp-viewport',
+		),
+		'editor' => array(
+			'jquery',
+			'lodash',
+			'wp-tinymce-lists',
+			'wp-a11y',
+			'wp-api-fetch',
+			'wp-blob',
+			'wp-blocks',
+			'wp-components',
+			'wp-compose',
+			'wp-core-data',
+			'wp-data',
+			'wp-date',
+			'wp-deprecated',
+			'wp-dom',
+			'wp-element',
+			'wp-hooks',
+			'wp-html-entities',
+			'wp-i18n',
+			'wp-is-shallow-equal',
+			'wp-keycodes',
+			'wp-notices',
+			'wp-nux',
+			'wp-polyfill',
+			'wp-rich-text',
+			'wp-tinymce',
+			'wp-token-list',
+			'wp-url',
+			'wp-viewport',
+			'wp-wordcount',
+		),
+		'element' => array( 'wp-polyfill', 'react', 'react-dom', 'lodash', 'wp-escape-html' ),
+		'escape-html' => array( 'wp-polyfill' ),
+		'format-library' => array(
+			'wp-components',
+			'wp-dom',
+			'wp-editor',
+			'wp-element',
+			'wp-i18n',
+			'wp-keycodes',
+			'wp-polyfill',
+			'wp-rich-text',
+			'wp-url',
+		),
+		'hooks' => array( 'wp-polyfill' ),
+		'html-entities' => array( 'wp-polyfill' ),
+		'i18n' => array( 'wp-polyfill' ),
+		'is-shallow-equal' => array( 'wp-polyfill' ),
+		'keycodes' => array( 'lodash', 'wp-polyfill' ),
+		'list-reusable-blocks' => array(
+			'lodash',
+			'wp-api-fetch',
+			'wp-components',
+			'wp-compose',
+			'wp-element',
+			'wp-i18n',
+			'wp-polyfill',
+		),
+		'notices' => array(
+			'lodash',
+			'wp-a11y',
+			'wp-data',
+			'wp-polyfill',
+		),
+		'nux' => array(
+			'lodash',
+			'wp-element',
+			'wp-components',
+			'wp-compose',
+			'wp-data',
+			'wp-deprecated',
+			'wp-i18n',
+			'wp-polyfill',
+		),
+		'plugins' => array( 'lodash', 'wp-compose', 'wp-element', 'wp-hooks', 'wp-polyfill' ),
+		'redux-routine' => array( 'wp-polyfill' ),
+		'rich-text' => array(
+			'lodash',
+			'wp-data',
+			'wp-deprecated',
+			'wp-escape-html',
+			'wp-polyfill',
+		),
```

Comment:
Noticing that 5.0's `rich-text` still includes `wp-blocks` as a dependency.

Reply:
Yes, and you can find `wp-polyfill-ecmascript` somewhere there, too. This PR tries to have an easy way to keep those 2 in sync.